test/table/test_table.py

```diff
@@ -194,7 +202,7 @@ def test_table_with_multiline_cells_and_without_headings(tmp_path):
     pdf = FPDF()
     pdf.add_page()
     pdf.set_font("Times", size=16)
-    with pdf.table(first_row_as_headings=False) as table:
+    with pdf.table(first_row_as_headings=False, padding=0) as table:
```

Comment:
The unit tests are there to ensure backward compatibility.
Hence, they should not be altered unless you have a very good reason to do so.

I think we should try to have those unit tests still passing when including the changes introduced by this PR

Reply:
changed default value, padding=0 no longer needed here